src/ripple/test/jtx/JTx.h

```diff
@@ -56,12 +59,99 @@ struct JTx
     {
     }
 
+#if defined(_MSC_VER) && _MSC_VER <= 1800
```

Comment:
As written, you pay for a copy construction and a move construction for the copy constructor and move constructor respectively. With your suggestion you pay for a copy construction, and a move then copy construction respectively. Or am I missing something?